src/idd/idd.utils.js

```diff
@@ -358,5 +360,16 @@ InteractiveDataDisplay.Utils =
                 return 11;
             else
                 return 0; //It is not IE
+        },
+
+        lastUsedIntId: 0,
+
+        getLastUsedIntId: function () {
```

Comment:
@lucymukh I'd rather use more informative name like "GetNextIntId()"
As getLastUsedIntId confuses, the name does not imply generation of new ID.